csharp/Platform.Collections/Segments/Walkers/AllSegmentsWalkerBase[T].cs

```diff
@@ -6,36 +6,32 @@
 namespace Platform.Collections.Segments.Walkers
 {
     /// <summary>
-    /// <para>
-    /// Represents the all segments walker base.
-    /// </para>
-    /// <para></para>
+    /// <para>Represents the base abstract class for walkers on all elements.</para>
+    /// <para>Представляет базовый абстрактный класс для проходчиков по всем элементам.</para>
     /// </summary>
     /// <seealso cref="AllSegmentsWalkerBase{T, Segment{T}}"/>
     public abstract class AllSegmentsWalkerBase<T> : AllSegmentsWalkerBase<T, Segment<T>>
     {
         /// <summary>
-        /// <para>
-        /// Creates the segment using the specified elements.
-        /// </para>
-        /// <para></para>
+        /// <para>Initializes a new instance of the <see cref="Segment"/> class.</para>
+        /// <para>Инициализирует новый экземпляр класса <see cref="Segment"/></para>
```

Comment:
```suggestion
        /// <para>Создаёт новый экземпляр класса <see cref="Segment"/></para>
```
`Create` --> `Создавать` (порождать)